pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -93,8 +97,13 @@ export const LogTable: React.FC<LogTableProps> = ({ entityId }) => {
                           field: "requestMethod",
                         },
                         {
-                          headerName: "Response time (seconds)",
+                          headerName: "Response time",
                           field: "responseTime",
+                          renderCell: () => {
+                            return (
+                              `${log?.responseTime.toString()}ms (${millisecondsToSeconds(log?.responseTime).toString()}s)`
```

Comment:
What would happen when `log` does not exist here?

I think it will be printing "mss" because the "ms" and "s" are not optional with the `log`.

Maybe something like `log && '${log.responseTime}ms (${msToSeconds(log.responseTime)})'`

Note that I also removed the .toString here, I think it works fine.